adsingestp/parsers/adsfeedback.py

```diff
@@ -0,0 +1,83 @@
+import json
+import logging
+
+from adsingestp import utils
+from adsingestp.parsers.base import BaseBeautifulSoupParser
+
+logger = logging.getLogger(__name__)
+
+
+class ADSFeedbackParser(BaseBeautifulSoupParser):
+    def __init__(self, json_string):
+        self.data = json.loads(json_string)
+
+    def parse(self, **kwargs):
```

Comment:
Again, to be clear, something like `def parse(self, data=None, **kwargs)` followed by `if data: self.data = data`